<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="box_main" native="true">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 221, 221);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="container_left">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(14, 25, 36);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="box_logo">
            <property name="maximumSize">
             <size>
              <width>351</width>
              <height>351</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../assets/logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="container_right" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QFrame" name="box_input">
            <property name="maximumSize">
             <size>
              <width>351</width>
              <height>461</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_title">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>10</y>
               <width>131</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>26</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>LOGIN</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_account">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>135</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Conta</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_pass">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>205</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Senha</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_account">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>160</y>
               <width>251</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_submit">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>300</y>
               <width>93</width>
               <height>28</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Entrar</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_new_account">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>380</y>
               <width>241</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>NÃ£o tenho conta</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_pass">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>230</y>
               <width>251</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
